src/GrpcProtos/capture.proto

```diff
@@ -48,6 +48,9 @@ message CaptureOptions {
   bool enable_introspection = 9;
 
   uint64 max_local_marker_depth_per_command_buffer = 10;
+
+  bool collect_memory_info = 11;
+  int64 memory_sampling_period_ns = 12;
```

Comment:
unit64?

Reply:
Change accordingly. Thanks so much for finding this!